### ISSUE TITLE
[Feature request] Tab folder support for Webdock list 

### ISSUE THREAD

**Issue by lens0021** [wavebox/waveboxapp#1414]
* Wavebox Version: 10.113.11-2 stable
Install Method: rpm
Wire Config: 1.2.15
Chromium: 113.0.5672.63
OS: linux/undefined
* Operating System & Version: Fedora workstation 38

### Bug/Feature description

Currently, Webdock list does not seem to support the tab folders feature:

![image](https://user-images.githubusercontent.com/28209361/236595420-e4a896a7-b757-42cd-a4a3-73a330c81f86.png)

While Webdock explorer do:

![image](https://user-images.githubusercontent.com/28209361/236595433-0ff17652-56f5-4e25-abae-3d212ea74272.png)

### Steps to reproduce

1. Create a tab folder in Webdock explorer view
2. Switch to Webdock list view

**Comment by Thomas101** [comment]
This is correct. We only support folders when using the Webdock explorer. We found that when trying to show lots of nested folders in the tab strip, it made the UI cluttered and confusing and meant that trying to navigate around involved lots of extra clicks or hidden menus that just didn't work well from a UX perspective.

This is why they're only displayed when using the Webdock explorer as there's enough space to show everything. It's worth noting that wwitching between the two modes doesn't lose your folder structure, so when you switch back it's still there.

**Comment by lens0021** [comment]
Thank you for clarifying!
